bentoml/_internal/frameworks/transformers.py

```diff
@@ -1,86 +1,43 @@
-import re
+from __future__ import annotations
+
+import os
 import typing as t
 import logging
+import importlib
 import importlib.util
 from typing import TYPE_CHECKING
-from importlib import import_module
 
-from simple_di import inject
-from simple_di import Provide
+import attr
+import attrs
```

Comment:
you only need to import `attr` here. `attrs` is a newer api that does route everything back to `attr`.
```suggestion

```